### PR TITLE
some 4554 support

### PR DESCRIPTION
**Pull request by hobokenn**
image

### CODE REVIEW

File: src/hooks/engine.rs [hunk 1]
```diff
@@ -154,7 +168,7 @@ pub static Host_InitializeGameDLL: Pointer<unsafe extern "C" fn()> = Pointer::em
     // Alternatively, find LoadEntityDLLs() and go to the parent function.
     Patterns(&[
         // 6153
-        pattern!(E8 ?? ?? ?? ?? 8B 0D ?? ?? ?? ?? 33 C0),
+        pattern!(E8 ?? ?? ?? ?? 8B 0D ?? ?? ?? ?? 33 C0 83 F9 01),
```
Comment (YaLTeR) [inline src/hooks/engine.rs:171]: Was it non-unique in 4554?

Comment (hobokenn) [reply src/hooks/engine.rs:171]: yes, it was

Comment (YaLTeR) [reply src/hooks/engine.rs:171]: Interesting, I wonder what the other function was.

**Comment by YaLTeR** [comment]
Thanks, this is awesome!

File: src/hooks/engine.rs (hunk 1 above)
Comment (hobokenn) [reply src/hooks/engine.rs:171]: one was `SV_WriteClientdataToMessage` and the other one was `VideoMode_SwitchMode`, this is what it matched for the latter one 
image

Comment (YaLTeR) [reply src/hooks/engine.rs:171]: I see, okay.

**Pull request merged**